src/components/WebChat/WebChat.css

```diff
@@ -0,0 +1,130 @@
+.webchat-container{
+  display: flex; 
+  flex-direction: column;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  bottom: 20px;
+  box-shadow: 5px 5px 20px #fff;
+}
+
+.webchat-panel-inner{
+  flex-direction: column;
+}
+
+.webchat-bar, .webchat-open__btn{
+  height: 45px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  background-color:#6e267b;
+  padding: 40px;
+}
+
+.webchat-bar-spacing, .webchat-btn-spacing{
```

Comment:
i can see that you are trying to manage spacing here, but line 22 also has padding applied. why?